includes/Plugin_Context.php

```diff
@@ -0,0 +1,17 @@
+<?php
+/**
+ * Class WordPress\Plugin_Check\Plugin_Context
+ *
+ * @package plugin-check
+ */
+
+namespace WordPress\Plugin_Check;
+
+/**
+ * Class representing the context in which the plugin is running.
+ *
+ * @since 0.1.0
```

Comment:
Maybe use `n.e.x.t` throughout, like we do in Performance Lab? Since we don't know what the first version will be.